lib/core/cursor.js

```diff
@@ -736,7 +736,8 @@ Cursor.prototype._initializeCursor = function(callback) {
       return;
     }
 
-    server.command(cursor.ns, cursor.cmd, cursor.options, queryCallback);
+    const commandOptions = Object.assign({}, cursor.options, cursor.cursorState);
```

Comment:
Does everything on cursorState belong in the command options?

Reply:
This is unclear. I just needed the `session` if it was set, so I changed the code to reflect that. Not to do too much hand waving in these reviews, but this is another thing that will just go away once we are explicitly using operations within the cursor. We won't ever be calling methods on a `Server` or `Connection`, that will happen in the `AggregationOperation.execute` or `FindOperation.execute` methods.